### PR TITLE
Add Dockerfile

### PR DESCRIPTION
**Pull request by scjody**
@chriddyp @BRONSOLO Please review

This allows dashboards.ly to run in its own Docker container as well as on Heroku.

Part of https://github.com/plotly/streambed/issues/5018

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+    apt-get -y install python python-pip libpq-dev python-dev && \
+    apt-get clean
+
+COPY init_db.py requirements.txt runserver.py /opt/dashboardsly/
+COPY dashboardsly /opt/dashboardsly/dashboardsly/
+
+RUN cd /opt/dashboardsly && pip install -r requirements.txt
+
+EXPOSE 80
+CMD cd /opt/dashboardsly; gunicorn runserver:app --log-file=- -b 0.0.0.0:80
```
Comment (BRONSOLO) [inline Dockerfile]: newline?

**Comment by BRONSOLO** [comment]
:dancer:

File: Dockerfile (hunk 1 above)
Comment (scjody) [inline Dockerfile]: Yeah.  And I've had a stern talk to my editor about doing this automatically in future.

Comment (BRONSOLO) [inline Dockerfile]: : )

**Comment by chriddyp** [comment]
cool!

**Pull request merged**